lib/src/icons/Carat-Down.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import './Carat-Down.scss';
```

Comment:
```suggestion
import './CaratDown.scss';
```

Going further, we don't use hyphens in any of our filenames.  I don't think we should start here.